documentation/onboarding.md

```diff
@@ -0,0 +1,152 @@
+# Onboarding guide
+
+The Agenda application connects to external scheduling applications like Microsoft Outlook Calendar, Google Calendar, or Zoom, and merges their independent meeting data into one place. By consolidating team member availability into one simplified workspace, you can quickly schedule meetings that work well for everyone. 
+
+This document guides you through the installation process on your device, creating a new account, and linking an external calendar.
+
+## System requirements
+
+The sections below outline the system requirements for Agenda depending on whether you are using [Windows] or [MacOS].
+
+### Windows system requirements
+
+Below is a list of hardware requirements for installing the Agenda application on a Windows machine:
+
+| Component | Requirement |
+| --- | --- |
+| Operating system | Windows 10 or Windows 11 |
+| Processor | Minimum 1.1 GHz |
+| Memory | 4.0 GB RAM |
+| Hard disk | 3.0 GB of available disk space |
+
+### MacOS system requirements
+
+Below are a list of hardware requirements for installing the Agenda application on a Mac machine:
+
+| Component | Requirement |
+| --- | --- |
+| Operating system | MacOS Catalina or newer |
+| Processor | 2.6 GHz 6-Core Intel Core i7 |
+| Memory | 4.0 GB RAM |
+| Hard disk | 1.5 GB of available disk space |
+
+## Installation
+
+Below are the installation steps for:
+* [Windows](#windows)
+* [MacOS](#mac)
+
+### Windows install {#windows}
```

Comment:
Same thing here re: headings inheriting their parent's context. The heading can just be "Windows", though the anchor should be something like #windows-install so we can distinguish between this and the other "Windows" H3 for system requirements.